ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/ros2/RDXROS2RobotVisualizer.java

```diff
@@ -92,15 +81,26 @@ public RDXROS2RobotVisualizer(RDXBaseUI baseUI,
       footstepHistoryGraphic.setColor(RobotSide.RIGHT, Color.SKY);
    }
 
+   public void setupCameraTracking(RDXFocusBasedCamera cameraForTracking)
+   {
+      this.cameraForTracking = cameraForTracking;
+      trackRobot.set(true);
+   }
+
+   public void createAndSetupStandalone(RDXBaseUI baseUI)
```

Comment:
This is a method I really wanted for the behavior test visualizer I'm working on.